modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/layout/admin/util/CPDefinitionSitemapURLProvider.java

```diff
@@ -161,13 +168,35 @@ protected void visitLayout(
 				_portal.getClassNameId(CProduct.class),
 				cpDefinition.getCProductId());
 
+		List<FriendlyURLEntryLocalization> friendlyURLEntryLocalizations =
+			_friendlyURLEntryLocalService.getFriendlyURLEntryLocalizations(
+				friendlyURLEntry.getFriendlyURLEntryId());
+
 		String productFriendlyURL =
 			currentSiteURL + urlSeparator +
 				friendlyURLEntry.getUrlTitle(themeDisplay.getLanguageId());
 
-		_sitemap.addURLElement(
-			element, productFriendlyURL, typeSettingsUnicodeProperties,
-			layout.getModifiedDate(), productFriendlyURL, null);
+		Map<Locale, String> alternateFriendlyURLs = new HashMap<>();
+
+		for (FriendlyURLEntryLocalization friendlyURLEntryLocalization :
+				friendlyURLEntryLocalizations) {
+
+			String alternateFriendlyURL =
+				currentSiteURL + urlSeparator +
+					friendlyURLEntryLocalization.getUrlTitle();
```

Comment:
why you didn't use the StringBundler?

Reply:
Hi @andrea-ale-sbarra , I was just following existing format in the code. I can update to StringBundler.

Reply:
Made the changes and updated PR.